README.md

```diff
@@ -1,33 +1,46 @@
 # patreon-php
+
 Interact with the Patreon API via OAuth.
 
+## Important notice about updating to 1.0.0 from earlier versions
+
+Patreon PHP library version 1.0.0 moves on to Patreon's v2 API, which is not compatible with old v1 calls. It also removes Art4 JSON library. Therefore directly upgrading from older versions to 1.0.0 would break compatibility of your installation. API v1 is going to be deprecated probably within 2019, so its important to get your integration compatible with API v2. Transition is easy - just translate your v1 calls to v2 calls using the documentation, and have your oAuth authorization include scopes. With API v2, you can only get access to the scopes you requested during authorization, and you need to ask for the data you want through includes. 
```

Comment:
Please change to `APIv1 will deprecated some time soon after APIv2 come out of beta, so its important to get your integration compatible with API v2`

Please remove `Transition is easy - just translate your v1 calls to v2 calls using the documentation, and have your oAuth authorization include scopes.`

Reply:
Done and gone.